gulpfile.js

```diff
@@ -211,8 +203,10 @@ gulp.task("watch", ["serve"], function () {
     gulp.watch(
         [
             path.join(ROOT_DIR, "**", "*.yml"),
-            path.join(SOURCE_DIR, "**", "*.html"),
-            path.join(SOURCE_DIR, "**", "*.md"),
+            path.join(SOURCE_DIR, "**", "*.html") + "!" + path.join(DOCS_DIR, "**"),
+            path.join(SOURCE_DIR, "**", "*.md") + "!" + path.join(DOCS_DIR, "**") ,
+            path.join(DOCS_DIR, "en", "dev", "**", "*.md") ,
+            path.join(DOCS_DIR, "en", "dev", "**", "*.html"),
```

Comment:
I didn't realise that although the docs folder is excluded, the English folder is included explicitly. Could you please add a comment to draw attention to these two includes?